timApp/static/scripts/tim/item/manageCtrl.ts

```diff
@@ -271,6 +271,12 @@ export class PermCtrl implements IController {
             await this.getTranslations();
             if (tr.id === this.item.id) {
                 this.syncTitle(tr.title);
+                // syncTitle does not update the current document path,
+                // force page reload manually with the new translation path
+                const new_path = tr.path
+                    .substring(0, tr.path.lastIndexOf(tr.old_langid))
```

Comment:
`tr.old_langid` saattaa olla tyhjä jono jos dokumentille ei ollut ennestään laitettu dokumentti-IDtä. Esimerkiksi <https://timdevs01-5.it.jyu.fi/manage/users/dz-dz/test-tr-rename>, eli lopusta puuttuu `/`. Eli silloin `new_path` olisi `https://timdevs01-5.it.jyu.fi/manage/users/dz-dz/test-tr-renamefi` (jos uusi kielikoodi olisi `fi`). Ainakin tuossa timdevs01-5ssa jos nyt asettaa kielikoodin, niin sivu virkistyy 404:een.

Eli pitänee käsitellä tuo tilanne `tr.old_langid` on tyhjä jono vähän eri tavalla jotta lopullinen polku olisi oikein.